src/com/dotcms/rest/api/v1/sites/ruleengine/rules/RuleResource.java

```diff
@@ -226,8 +219,19 @@ private User getUser(@Context HttpServletRequest request) {
 
     private List<RestRule> getRulesInternal(User user, Ruleable host) {
         try {
+            List<RestRule> restRules = new ArrayList<>();
+
             List<Rule> rules = rulesAPI.getAllRulesByParent(host, user, false);
-            return rules.stream().map(ruleTransform.appToRestFn()).collect(Collectors.toList());
+            for (Rule rule : rules) {
+                try{
+                    restRules.add(ruleTransform.appToRest(rule, user));
+                } catch (Exception transEx) {
+                    String ruleName = UtilMethods.isSet(rule.getName()) ? rule.getName() : "N/A";
+                    Logger.error(this, "Error parsing Rule named: " + ruleName + " to ReST");
```

Comment:
Make sure not to swallow the stack trace. Also, just let the rule print its own debug via the tostring method:
`Logger.error(this, ex, "Error transforming Rule: " + rule);`